moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1953,6 +1985,48 @@ double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* gro
   return last_valid_percentage;
 }
 
+double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
+                                                      const LinkModel* link, const Eigen::Affine3d& target,
+                                                      bool global_reference_frame, double max_step,
+                                                      double jump_threshold,
+                                                      const GroupStateValidityCallbackFn& validCallback,
+                                                      const kinematics::KinematicsQueryOptions& options)
+{
+  jump_threshold_t jt;
+  jt.jump_threshold_factor = jump_threshold;
+  jt.prismatic_jump_threshold = 0.0;
+  jt.revolute_jump_threshold = 0.0;
+  return computeCartesianPath(group, traj, link, target, global_reference_frame, max_step, jt, validCallback, options);
+}
+
+double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
+                                                      const LinkModel* link, const Eigen::Affine3d& target,
+                                                      bool global_reference_frame, double max_step,
+                                                      double revolute_jump_threshold, double prismatic_jump_threshold,
+                                                      const GroupStateValidityCallbackFn& validCallback,
+                                                      const kinematics::KinematicsQueryOptions& options)
+{
+  jump_threshold_t jt;
+  jt.jump_threshold_factor = 0.0;
+  jt.prismatic_jump_threshold = prismatic_jump_threshold;
+  jt.revolute_jump_threshold = revolute_jump_threshold;
+  return computeCartesianPath(group, traj, link, target, global_reference_frame, max_step, jt, validCallback, options);
+}
+
+double moveit::core::RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
+                                                    const moveit::core::jump_threshold_t& jump_threshold)
+{
+  double percentage = 1.0;
+  if (jump_threshold.jump_threshold_factor > 0.0)
+    percentage *= testJointSpaceJump(group, traj, jump_threshold.jump_threshold_factor);
+
+  if (jump_threshold.prismatic_jump_threshold > 0.0 || jump_threshold.revolute_jump_threshold > 0.0)
+    percentage *= testJointSpaceJump(group, traj, jump_threshold.revolute_jump_threshold,
+                                     jump_threshold.prismatic_jump_threshold);
```

Comment:
add ``else``
(none are set)
throw error (testJointStateJump should not have been called)